app/controllers/oauth/applications_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Oauth::ApplicationsController < Doorkeeper::ApplicationsController
+  before_action :authenticate_user!
+
+  def index
+    @applications = current_user.oauth_applications
+  end
+
+  # only needed if each application must have some owner
+  def create
+    @application = Doorkeeper::Application.new(application_params)
+    @application.owner = current_user if Doorkeeper.configuration.confirm_application_owner?
+    if @application.save
+      flash[:notice] = I18n.t(:notice, :scope => [:doorkeeper, :flash, :applications, :create])
+      redirect_to oauth_application_url(@application)
+    else
+byebug
```

Comment:
Oops. Looks like a stray debugger was left in. This was on the sad path, so easy to miss.